RNTester/RNTesterUnitTests/RCTImageUtilTests.m

```diff
@@ -10,7 +10,11 @@
 #import <Foundation/Foundation.h>
 #import <XCTest/XCTest.h>
 
+#if __has_include(<React/RCTImageUtils.h>) // TODO(macOS ISS#2323203)
```

Comment:
The Pod version of the build and the static xcode version have different header include paths for RCTImageUtils.h.   Post .60 upstream has refactored the location of this header and this delta can go away after we've merged.